packages/EasyTest/src/Bridge/PhpUnit/PHPUnitExtension.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyTest\Bridge\PhpUnit;
+
+use EonX\EasyTest\Bridge\Symfony\Mailer\EventListener\MessageLoggerListenerStub;
+use PHPUnit\Runner\AfterTestHook;
+
+class PHPUnitExtension implements AfterTestHook
+{
+    public function executeAfterTest(string $test, float $time): void
+    {
+        // We will do this reset even if MessageLoggerListenerStub is not enabled.
+        // This is faster because we don't need to boot the kernel to get the bundle config.
+        MessageLoggerListenerStub::reset();
```

Comment:
In the project, it was done in the `setUp`. Here is in `executeAfterTest`. No difference? Why not `executeBeforeTest`?

Reply:
`tearDown` methods don't execute if a test fails with an Exception.

PhpUnit executes extensions, even if a test fails with an Exception.